### DISCUSSION TITLE
Tutorial for PyTorch QAT

### DISCUSSION THREAD

**Discussion by ram95014** [sony/model_optimization#1072] [Ideas]
### Issue Type

Others

### Source

source

### MCT Version

2.0.0

### OS Platform and Distribution

Any

### Python version

3.9

### Describe the issue

```shell
Eventhough QAT is an experimental feature, it would be useful for developers to have a tutorial on how to use MCT QAT with PyTorch. We have one for Keras, but the PyTorch one would be useful too. Please consider adding a PyTorch based QAT tutorial.
```

### Expected behaviour

```shell
N.A
```

### Code to reproduce the issue

```shell
N.A
```

### Log output

```shell
N.A
```

**Comment by ServiAmirPM** [comment]
Thanks for the suggestion - great idea. 
That is on our roadmap. We will update you once this gets into production ;)
